doc/release-notes.md

```diff
@@ -145,6 +145,8 @@ Tools and Utilities
   like `-onlynet=<network>` or to upgrade to current and future Tor releases
   that support Tor v3 addresses only.  (#21595)
 
+- Update `-getinfo` to return data in user-friendly format that also reduces vertical space. (#21832)
```

Comment:
```suggestion
- Update `-getinfo` to return data in a user-friendly format that also reduces vertical space. (#21832)
```

Reply:
👍 Resolved it in [b634ff2](https://github.com/bitcoin/bitcoin/commit/b634ff26ab8cf4bdffed1a75300ec63a9da22cf1)